server/src/main/java/org/elasticsearch/search/sort/FieldSortBuilder.java

```diff
@@ -273,6 +286,22 @@ public FieldSortBuilder setNumericType(String numericType) {
         return this;
     }
 
+    /**
+     * Returns the external format that is specified via {@link #setFormat(String)}
+     */
+    String getFormat() {
```

Comment:
Should it be public ?

Reply:
fixed in https://github.com/elastic/elasticsearch/pull/70357/commits/3fa2ab6e7acbaf6e734767ceea028eeea82efb70